var/spack/repos/builtin/packages/doxygen/package.py

```diff
@@ -50,7 +50,7 @@ class Doxygen(Package):
     depends_on("graphviz", when="+graphviz", type='run')
 
     def install(self, spec, prefix):
-        cmake('.', *std_cmake_args)
```

Comment:
are these changes necessary? I think it has been fix a while ago. If so, please remove all 
`cmake() -> which('cmake')()` from this PR so that it's easier to review.